service/src/test/resources/ehr/mapper/diagnosticreport/fhir_bundle.json

```diff
@@ -288,6 +288,51 @@
         ]
       }
     },
+    {
+      "resource":{
+        "resourceType":"Observation",
+        "id":"FILING-COMMENT-WITH-NO-COMMENT",
+        "meta":{
+          "profile":[
+            "https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-Observation-1"
+          ]
+        },
+        "identifier":[
+          {
+            "system":"https://EMISWeb/A82038",
+            "value":"FILING-COMMENT-WITH-NO-COMMENT"
+          }
+        ],
+        "status":"unknown",
+        "code":{
+          "coding":[
+            {
+              "system":"http://snomed.info/sct",
+              "code":"37331000000100",
+              "display":"Comment note"
+            }
+          ]
+        },
+        "subject":{
+          "reference":"Patient/DAED5527-1985-45D9-993E-C5FF51F36828"
+        },
+        "effectiveDateTime":"2020-12-15T15:17:04.917+00:00",
+        "issued":"2020-12-15T15:17:04.917+00:00",
+        "performer":[
+          {
+            "reference":"Practitioner/C8FD0E2C-3124-4C72-AC8D-ABEA65537D1B"
+          }
+        ],
+        "related":[
+          {
+            "type":"derived-from",
+            "target":{
+              "reference":"Observation/461839A4-9B28-46D4-BC27-297C46F4F80E"
+            }
+          }
+        ]
```

Comment:
Docs for filing comment related:
> Reference to the test result or test group header that the filing comments resource relates to. Where the filing comments relate to the test report, the reference is made from the ‘Test report’ to the filing comment only.

I think we should remove this derived-from, to be a bit more realistic to the spec.

Reply:
Agreed!